Client/Coordinators/LibraryCoordinator.swift

```diff
@@ -15,7 +15,7 @@ class LibraryCoordinator: BaseCoordinator, LibraryPanelDelegate {
     private let profile: Profile
     private let tabManager: TabManager
     private let libraryViewController: LibraryViewController
-    weak var parentCoordinator: LibraryCoordinatorDelegate?
+    weak var parentCoordinator: (LibraryCoordinatorDelegate & LibraryPanelDelegate)?
 
```

Comment:
^ This might look strange and I agree. The way it is implemented is to delegate back to the browser view controller, how we did with the deeplinks.

Reply:
Oh I see! I might prefer having this on the protocol declaration instead then? Would that make sense?
```
protocol LibraryCoordinatorDelegate: AnyObject, LibraryPanelDelegate {
```